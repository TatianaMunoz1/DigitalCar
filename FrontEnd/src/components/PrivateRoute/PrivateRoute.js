import { Route, Redirect } from "react-router-dom";

export default function PrivateRoute(props) {
    return props.auth ? <Route {...props} /> : <Redirect to="/" />
}