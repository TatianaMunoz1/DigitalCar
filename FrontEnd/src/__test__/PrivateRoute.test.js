import React from "react";
import { mount, shallow } from "enzyme";
import "@testing-library/jest-dom/extend-expect";
import { render, screen } from "@testing-library/react";
import { Router } from "react-router-dom";
import { createMemoryHistory } from "history";
import PrivateRoute from "../components/PrivateRoute/PrivateRoute";



describe("Private Route", () => {

    test("render  Private Route ", () => {
        const wrapper = shallow(<PrivateRoute/>)
        expect(wrapper).toMatchSnapshot();
    });
});