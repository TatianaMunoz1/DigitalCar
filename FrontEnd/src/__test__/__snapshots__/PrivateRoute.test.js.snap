// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Private Route render  Private Route  1`] = `ShallowWrapper {}`;
